sw/device/lib/dif/dif_kmac_unittest.cc

```diff
@@ -585,4 +596,130 @@ TEST_F(KmacEndTest, Error) {
   op_state_.squeezing = false;
   EXPECT_EQ(dif_kmac_end(&kmac_, &op_state_), kDifError);
 }
+
+class KmacConfigureTest : public KmacTest {
+ protected:
+  dif_kmac_config_t kmacConfig_ = {
```

Comment:
Variables are snake case, so this should be `kmac_config_`.

Reply:
Done